executable/Main.hs

```diff
@@ -1,6 +1,10 @@
-module Main where
+module Main (main) where
 
-import qualified HelloWorld
+import qualified API
+
+import qualified Network.Wai.Handler.Warp as Warp
 
 main :: IO ()
-main = HelloWorld.helloWorld
+main = do
```

Comment:
Why did you decide to put this in the executable rather than the library?

Reply:
I felt like it looked better coming from main, rather than just passing it to some library.

Reply:
I am good either way though!